packages/sdk/src/conversation/interface.ts

```diff
@@ -163,6 +171,36 @@ export interface TeamsFxBotCommandHandler {
   ): Promise<string | Partial<Activity> | void>;
 }
 
+/**
+ * Interface for a command handler that can process sso command to a TeamsFx bot and return a response.
+ */
+export interface TeamsFxBotSsoCommandHandler {
+  /**
+   * command id used to create sso command dialog, if not assigned, it will generate random command id
+   */
+  commandId?: string;
+
+  /**
+   * The string or regular expression patterns that can trigger this handler.
+   */
+  triggerPatterns: TriggerPatterns;
+
+  /**
+   * Handles a bot command received activity.
+   *
+   * @param context The bot context.
+   * @param message The command message the user types from Teams.
+   * @param ssoToken The sso token which can be used to exchange access token for the bot.
+   * @returns A `Promise` representing an activity or text to send as the command response.
+   * Or no return value if developers want to send the response activity by themselves in this method.
+   */
+  handleCommandReceived(
```

Comment:
can we change `ssoToken: string` to `token: TeamsSsoPromptTokenResponse` which contains bot sso token and access token?

Reply:
Sure